src/main/java/com/dev/movietheatre/model/MovieSession.java

```diff
@@ -16,7 +16,7 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    @ManyToOne (fetch = FetchType.LAZY)
+    @ManyToOne
     @JoinColumn(name = "movie_id")
     private Movie movie;
     @ManyToOne (fetch = FetchType.LAZY)
```

Comment:
what about this?

Reply:
this fetch lefted from last hw, it's not needed anymore, but it created LazyIntitializationException in my MovieSession for Movie. so i deleted. Is't problem?

Reply:
no, this is not a problem, but what about cinemaHall?
my comment was left to line 22)

Reply:
i'll delete it also. I've not understand that comment was about line 22, at first.